modules/contrib_rules_jvm/0.21.2/presubmit.yml

```diff
@@ -0,0 +1,18 @@
+bcr_test_module:
+  matrix:
+    platform: ["debian10", "macos", "ubuntu2004", "windows"]
+  tasks:
+    gazelle:
```

Comment:
Given the CI failure, I'm not sure whether multiple test modules are supported.

cc @meteorcloudy

Reply:
I believe it actually could work. Just set `module_path: ""`, you can still configure different working_directory for different tasks like normal CI pipelines.
/cc @illicitonion

Reply:
That does indeed work! 

@illicitonion The remaining failure is Windows-only.